usaspending_api/references/v2/views/city.py

```diff
@@ -65,26 +65,23 @@ def prepare_search_terms(request_data):
     return [es_sanitize(field).upper() if isinstance(field, str) else field for field in fields]
 
 
-def create_elasticsearch_query(return_fields, scope, search_text, country, state):
-    query_string = create_es_search("wildcard", scope, search_text, country, state)
+def create_elasticsearch_query(return_fields, scope, search_text, country, state, limit):
+    query_string = create_es_search(scope, search_text, country, state)
```

Comment:
Super duper minor thing, but this is no longer returning a string.

Reply:
Done